.evergreen.yml

```diff
@@ -191,7 +191,7 @@ functions:
           # debug option is not very useful in normal runs, but helpful when
           # debugging any issues with tests. Set to "mocha*", "hadron*", or
           # "mongo*" for some helpful output from the test tooling we are using
-          DEBUG=${debug} MONGODB_VERSION=${mongodb_version|4} npm run --unsafe-perm test-ci -- --stream
+          DEBUG=${debug} MONGODB_VERSION=${mongodb_version|4} npm run --unsafe-perm test-ci -- --stream --ignore compass-e2e-tests
```

Comment:
Is this a change in behavior? Or do these e2e tests happen elsewhere now?

Reply:
Oh, yeah, sorry, I'll add a comment explaining this better. So we have a separate task called `test-packaged-app`, so this was just a duplication of the same test suite against a slightly different app build. The difference is that when run as part of `test-ci` it runs against a "local dev" version of the app, whereas `test-packaged-app` is running against a packaged application that we will eventually publish. So even though there might be slight difference in the results, the `test-packaged-app` is actually the one we are most interested in and taking into consideration that `test-ci` is the longest task now, we probably want to cut the running time for it as much as possible (we also do the same thing in GitHub actions)